inst/helper-objects-for-testing.R

```diff
@@ -0,0 +1,13 @@
+# code for the reference values in `testthat/helper-objects.R`:
+# expected parametric probability results form the rms package for row 1 of
+# the lung data at times 1, 500, and 1000 using weibull model
+
+library(rms)
+mod <- psm(Surv(time, status) ~ age + sex, data = lung, dist = "weibull")
+lambda <- Hazard(mod)
+survival_fun <- Survival(mod)
+lp <- mod$linear.predictors
+
+prob_times <- c(0, 500, 1000)
+rms_surv <- survival_fun(prob_times, lp[1])
+rms_haz <- lambda(prob_times, lp[1])
```

Comment:
So is the idea that these generate `rms_surv` and `rms_haz` objects, but you copy over the values that got generated here into the hardcoded values in `tests/testthat/helper-objects.R`?

If so, you might want to make this file generate data objects (like with `saveRDS()`) for `rms_surv` and `rms_haz` instead and save them in `tests/testthat/data/`.

Then `tests/testthat/helper-objects.R` can just do `readRDS()` to load them in, like:

```r
rms_surv <- readRDS(test_path("data", "rms_surv.rds"))
```

Or something like that.

I would go ahead and put in the change to `0` for the first value of `rms_haz` in this file too